compose/config/config.py

```diff
@@ -70,7 +70,8 @@
     'mem_limit',
     'memswap_limit',
     'net',
-    'oom_score_adj'
+    'oom_score_adj',
+    'oom_kill_disable',
```

Comment:
This is alarming - there was previously no comma after `'oom_score_adj'`. This suggests that test coverage is insufficient here.